libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/audio/AudioTypeChain.kt

```diff
@@ -0,0 +1,16 @@
+package com.mapbox.navigation.core.telemetry.audio
+
+internal class AudioTypeChain {
```

Comment:
Should this be moved to `libnavigation-util` module?

Reply:
👍 Sure.